android/expoview/src/main/java/host/exp/exponent/notifications/NotificationConstants.java

```diff
@@ -5,7 +5,7 @@
 public class NotificationConstants {
   public static final int MAX_COLLAPSED_NOTIFICATIONS = 5;
   public static final String NOTIFICATION_MESSAGE_KEY = "message"; // deprecated
-  public static final String NOTIFICATION_EXPERIENCE_ID_KEY = "experienceId";
+  public static final String NOTIFICATION_EXPERIENCE_ID_KEY = "experienceId"; // scopeKey
```

Comment:
Despite the awkward "key key", it seems more correct to rename this field:
```suggestion
  public static final String NOTIFICATION_SCOPE_KEY_KEY = "experienceId"; // scopeKey
```